centreon/src/Core/Infrastructure/Common/Repository/DbFactoryUtilitiesTrait.php

```diff
@@ -75,4 +75,21 @@ public static function getFloatOrNull($property): ?float
     {
         return ($property !== null) ? (float) $property : null;
     }
+
+    /**
+     * @param string|null $property
+     *
+     * @return int[]|array{}
+     */
+    public static function fromStringToArrayOfInts(?string $property): array
+    {
+        if ($property === null) {
```

Comment:
If the string is `""` in input, you will have a `[0]` in output.
I don't think this is what you want, :warning: about `explode`
```suggestion
        if ($property === null || $property === '') {
```